WordPress/Classes/Utility/Migrator/CoreDataIterativeMigrator.swift

```diff
@@ -75,24 +81,53 @@ class CoreDataIterativeMigrator: NSObject {
             modelsToMigrate = modelsToMigrate.reversed()
         }
 
+        // Nested function for retrieving a model's version name.
+        // Used to give more context to errors.
+        func versionNameForModel(model: NSManagedObjectModel) -> String {
```

Comment:
Does this need to be public?

Reply:
Well, this is a nested function with local scope. It actually can't be declared `public`, which makes sense as there would be no way for code outside of its containing method to call it.

Reply:
Arg! Typos! Thanks for spotting that!